### PR TITLE
Design master page

### PR DESCRIPTION
**Pull request by hoanghuetm**

### CODE REVIEW

**Comment by hoanghuetm** [comment]
ready

File: resources/views/client/layouts/master.blade.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <!-- Mobile specific metas  -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Favicon  -->
+    <link rel="shortcut icon" type="image/x-icon" href="{{ config('image.ico') }}">
+
+    <!-- Bootstrap css-->
+    <link rel="stylesheet" href="{{ asset('bower_components/client_bower/bootstrap/dist/css/bootstrap.min.css') }}">
+
+    <!-- Fontawesome css-->
```
Comment (huyenminnn) [inline resources/views/client/layouts/master.blade.php]: bỏ hết comment

**Comment by hoanghuetm** [comment]
ready

**Comment by quoctd-0607** [comment]
commented
conflict code rồi em

**Comment by hoanghuetm** [comment]
ready

**Pull request merged**